test/python/WMCore_t/Services_t/MicroService_t/Unified_t/Transferor_t.py

```diff
@@ -14,22 +14,26 @@
 from WMCore.Services.PhEDEx.PhEDEx import PhEDEx
 from WMCore.Services.MicroService.Unified.Transferor import \
         RequestStore, RequestManager
+from WMQuality.Emulators.PhEDExClient.MockPhEDExApi import MockPhEDExApi
+from WMQuality.Emulators.EmulatedUnitTestCase import EmulatedUnitTestCase
 
-
-class TransferorTest(unittest.TestCase):
+class TransferorTest(EmulatedUnitTestCase):
     "Unit test for Transferor module"
     def setUp(self):
         "init test class"
+        super(TransferorTest, self).setUp()
         self.group = 'DataOps'
         self.interval = 2
+        self.phedex = MockPhEDExApi()
         self.rmgr = RequestManager(group=self.group, interval=self.interval, verbose=True)
-        self.phedex = PhEDEx()
 
         # get some subscriptions from PhEDEx to play with
         data = self.phedex.subscriptions(group=self.group)
+        print("### data", data)
         for datasetInfo in data['phedex']['dataset']:
             dataset = datasetInfo.get('name')
-            print("### dataset info from phedex, #files %s" % datasetInfo.get('files', 0))
+            print("### dataset info from phedex, dataset %s #files %s" \
```

Comment:
Is there a reason to keep all these print statements in the test now that it is presumably working OK?